draft-ietf-quic-transport.md

```diff
@@ -3676,6 +3679,9 @@ INTERNAL_ERROR (0x1):
 : The endpoint encountered an internal error and cannot continue with the
   connection.
 
+SERVER_BUSY (0x2):
+: The server is currently busy and does not accept any new connections.
```

Comment:
Others have a line between entry and definition.  I don't particularly care for that style, but let's be consistent anyway.